packages/alfa-css/src/calculation/math-expression/expression.ts

```diff
@@ -4,7 +4,7 @@ import { Serializable } from "@siteimprove/alfa-json";
 import { None, Option } from "@siteimprove/alfa-option";
 
 import { Unit } from "../unit/unit";
-import { Length, Number, Numeric, Percentage } from "../numeric";
+import { Length, Number, Numeric, Percentage } from "../numeric/index";
```

Comment:
**question (non-blocking):** I think `index` can still be omitted, is it added on purpose?

Reply:
🙈 IDE messing up when moving and renaming files.